### PR TITLE
Update endpoint-dlp-using.md

### PR DESCRIPTION
**Pull request by shpalta**
@chrfox can you please review and commit this?

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/endpoint-dlp-using.md [hunk 1]
```diff
@@ -50,6 +50,11 @@ In advanced classification, content is sent from the local device to the cloud s
 
 If bandwidth utilization is not a concern, you can not set a limit and allow unlimited utilization.
 
+Support for advanced classification scanning and protection is currently available in the following Windows versions :
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-using.md]:
```suggestion
Support for advanced classification scanning and protection is currently available in the following Windows versions: 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shpalta, kindly commit the minor suggested change. Many thanks.

**Comment by garycentric** [comment]
@shpalta 
Microsoft-credentialed contributors are strongly encouraged to work in the private repos, not the public. The private repos provide checks for build integrity, checks for spelling & grammar, and provide preview links for added and modified articles. The private repos also have a shorter path to publication, and they're a secure place to add content prior to official release. 

For more information about working in the private repos, please see [Where do I start?](https://review.docs.microsoft.com/en-us/office-authoring-guide/office-docs-authoring-overview?branch=master#where-do-i-start) in the MARVEL authoring guide. If you have questions or need help, please contact [Ask an Admin](https://aka.ms/askanadmin) in Teams.

**Pull request merged**